src/winstanley/WdlAnnotator.scala

```diff
@@ -46,10 +48,10 @@ class WdlAnnotator extends Annotator {
     case value: WdlCallableLookup =>
       // TODO (Issue 63): Work out a better way to verify that imports exist and (if local?) that an appropriate task/workflow inside them exist.
       // For now to avoid error highlights, I'll just suppress errors for things that look like FQNs
-      if (value.getFullyQualifiedName.getNode.getChildren(null).length == 1) {
+      if (value.getFullyQualifiedName.getNode.getChildren(TokenSet.ANY).length == 1) {
```

Comment:
I'm pretty sure I've interpreted the meaning of `getChildren(null)` as "get children of any type", but could use a double-check

Reply:
I'm happy to let the tests passing be the judge of "is this valid", and it seems so.